src/components/Day/DayInfoSection/DayChange.tsx

```diff
@@ -13,8 +11,8 @@ import PrevArrow from '/public/assets/PrevArrow.png';
 
 function DayChange() {
   const router = useRouter();
-  const [dayDate, setDayDate] = useRecoilState(dayInfo);
-  const [dayChange, setDayChange] = useRecoilState(dayCount);
+  const [dayDate, setDayDate] = useState(getTodayDate(0));
+  const [dayChange, setDayChange] = useState(0);
```

Comment:
1. 이미 getTodayDate(0)함수를 호출해 day 변수에 담고있기 때문에 그걸 계속 사용해도 괜찮을 것 같아요!
2. 저 0의 의미가 무엇인지 불분명해서, 상수를 활용해도 좋을 것 같아요!
3. dayChage보다 dateChange가 좀 더 의미가 명확할 것 같아요! 

```suggestion
  const DEFAULT_DATE_CHANGE = 0;
  const day = getTodayDate(DEFAULT_DATE_CHANGE);
  const [dayDate, setDayDate] = useState(day);
  const [dateChange, setDateChange] = useState(DEFAULT_DATE_CHANGE);
```